### PR TITLE
make dependency resolution quieter

### PR DESCRIPTION
**Pull request by cmacdonald**

### CODE REVIEW

File: kernel-api/src/main/scala/org/apache/toree/dependencies/CoursierDependencyDownloader.scala [hunk 1]
```diff
@@ -148,8 +149,12 @@ class CoursierDependencyDownloader extends DependencyDownloader {
...
     // Print success
     val uris = localArtifacts.flatMap(_.toOption).map(_.toURI)
-    uris.map(_.getPath).foreach(p => printStream.println(s"-> New file at $p"))
-
+    
+    
+    if (verbose) uris.map(_.getPath).foreach(p => printStream.println(s"-> New file at $p"))
+    
+    printStream.println("Obtained " + uris.size() + " files")
+
```
Comment (lresende) [inline kernel-api/src/main/scala/org/apache/toree/dependencies/CoursierDependencyDownloader.scala]: I believe this should be: uris.size

**Comment by cmacdonald** [comment]
Bah, I did fix that - I dont know why the PR didn't give the final commit. Will try again :-(

**Comment by lresende** [comment]
@cmacdonald Any progress here?

**Comment by lresende** [comment]
LGTM, I fixed the `uris.size` locally so I can merge.

**Comment by lresende** [comment]
And Green Build status available at:
https://travis-ci.org/lresende/incubator-toree/builds/434411112

**Comment by cmacdonald** [comment]
Thanks @lresende - Sorry I didn't get back to this!

**Pull request closed without merging**